### PR TITLE
DSS-trying-to-find-undefined

### PR DESCRIPTION
**Pull request by GiCharkviani**
Fix this error: [GCP](https://console.cloud.google.com/errors/detail/COW0wtOD-5al8QE;filter=%5B%22undefined%22%5D;time=P30D?project=broad-ddp-test&supportedpurview=project)
And trying to find what is 'undefined'

### CODE REVIEW

File: ddp-workspace/projects/toolkit/src/lib/components/app/app-redesigned-base.component.ts [hunk 1]
```diff
@@ -39,20 +39,24 @@ export class AppRedesignedBaseComponent implements OnInit, OnDestroy {
     }
 
     private initMailingListDialogListener(): void {
-        const modalOpen = this.communicationService.openJoinDialog$.subscribe(() => {
-            this.dialog.open(JoinMailingListComponent, this.DIALOG_BASE_SETTINGS);
-        });
-        this.anchor.addNew(modalOpen);
+        if(this.communicationService) {
```
Comment (aweng98) [inline ddp-workspace/projects/toolkit/src/lib/components/app/app-redesigned-base.component.ts:42]: Is this file change related to error mentioned in PR comment?

**Review by aweng98 — APPROVED** [review]

File: ddp-workspace/projects/toolkit/src/lib/components/app/app-redesigned-base.component.ts (hunk 1 above)
Comment (GiCharkviani) [reply ddp-workspace/projects/toolkit/src/lib/components/app/app-redesigned-base.component.ts:42]: Yes
